public/app/plugins/datasource/cloud-monitoring/CloudMonitoringMetricFindQuery.ts

```diff
@@ -112,7 +112,7 @@ export default class CloudMonitoringMetricFindQuery {
     }
     const refId = 'handleResourceTypeQueryQueryType';
     const labels = await this.datasource.getLabels(selectedMetricType, refId, projectName);
-    return labels['resource.type'].map(this.toFindQueryResult);
+    return labels.hasOwnProperty('resource.type') ? labels['resource.type'].map(this.toFindQueryResult) : [];
```

Comment:
you could probably write this as `labels['resource.type']?.map() ?? []` if you so cared

Reply:
oh I didn't mean to actually submit this comment 😅

Reply:
yes yes much better, thanks!